idea/idea-core/src/org/jetbrains/kotlin/idea/core/util/PsiLinesUtils.kt

```diff
@@ -47,18 +49,26 @@ fun PsiElement.getLineCount(): Int {
     if (doc != null) {
         val spaceRange = textRange ?: TextRange.EMPTY_RANGE
 
-        if (spaceRange.endOffset <= doc.textLength) {
+        if (spaceRange.endOffset <= doc.textLength && spaceRange.startOffset < spaceRange.endOffset) {
             val startLine = doc.getLineNumber(spaceRange.startOffset)
             val endLine = doc.getLineNumber(spaceRange.endOffset)
 
-            return endLine - startLine
+            return endLine - startLine + 1
         }
     }
 
     return (text ?: "").count { it == '\n' } + 1
 }
 
-fun PsiElement.isMultiLine(): Boolean = getLineCount() > 1
+fun PsiElement.isMultiLine(): ThreeState {
```

Comment:
Why introduce `TreeState`? The `UNSURE` value will never occur

Reply:
Actually it supposed to be `?: return -1` in count lines but I missed it.

After thinking a little bit I removed `ThreeState` because it seems too verbose. Now I throw exception in case `text` is `null`